tests/features/arg_matching.py

```diff
@@ -73,167 +73,167 @@ def test_module_level_function_with_kwargs(self):
-                "  Expected call[3]:\tmethod1(arg1=<class 'str'>, arg2=<class 'int'>)"
+                "Arguments for call method did not match expectations:\n"
+                '  Received call:\tmethod("string", 12, 14)\n'
+                "  Expected call[1]:\tmethod(arg1=<class 'str'>, arg2=<class 'int'>)\n"
+                "  Expected call[2]:\tmethod(arg1=<class 'str'>, arg2=<class 'int'>)\n"
+                "  Expected call[3]:\tmethod(arg1=<class 'str'>, arg2=<class 'int'>)"
             ),
         ):
-            instance.method1("string", 12, 14)
+            instance.method("string", 12, 14)
 
     def test_flexmock_should_match_types_on_multiple_arguments_generic(self):
         class Foo:
-            def method1(self, a, b, c):  # pylint: disable=invalid-name
+            def method(self, a, b, c):  # pylint: disable=invalid-name
                 pass
 
         instance = Foo()
-        flexmock(instance).should_receive("method1").with_args(object, object, object).and_return(
+        flexmock(instance).should_receive("method").with_args(object, object, object).and_return(
             "ok"
         )
-        assert instance.method1("some string", None, 12) == "ok"
-        assert instance.method1((1,), None, 12) == "ok"
-        assert instance.method1(12, 14, []) == "ok"
-        assert instance.method1("some string", "another one", False) == "ok"
+        assert instance.method("some string", None, 12) == "ok"
+        assert instance.method((1,), None, 12) == "ok"
+        assert instance.method(12, 14, []) == "ok"
+        assert instance.method("some string", "another one", False) == "ok"
         with assert_raises(
             exceptions.MethodSignatureError,
             (
-                "Arguments for call method1 did not match expectations:\n"
-                '  Received call:\tmethod1("string", 12)\n'
-                "  Expected call[1]:\tmethod1(a=<class 'object'>, "
+                "Arguments for call method did not match expectations:\n"
+                '  Received call:\tmethod("string", 12)\n'
+                "  Expected call[1]:\tmethod(a=<class 'object'>, "
                 "b=<class 'object'>, c=<class 'object'>)"
             ),
         ):
-            instance.method1("string", 12)  # pylint: disable=no-value-for-parameter
+            instance.method("string", 12)  # pylint: disable=no-value-for-parameter
         flexmock_teardown()
-        flexmock(instance).should_receive("method1").with_args(object, object, object).and_return(
+        flexmock(instance).should_receive("method").with_args(object, object, object).and_return(
             "ok"
         )
         with assert_raises(
             exceptions.MethodSignatureError,
             (
-                "Arguments for call method1 did not match expectations:\n"
-                '  Received call:\tmethod1("string", 12, 13, 14)\n'
-                "  Expected call[1]:\tmethod1(a=<class 'object'>, "
```

Comment:
BTW this change with both actual and expected call here has been very useful at work :relieved: